docker/grafana/provisioning/dashboards/services-all.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: 1
+
+providers:
+  - name: Services-Dashboard    # A uniquely identifiable name for the provider
+    folder: Services # The folder where to place the dashboards
+    type: file
```

Comment:
Importing individual files is not scalable and should be handled by a folder based import provisioner as shown here: https://github.com/hashgraph/hedera-local-node/blob/main/compose-network/grafana/dashboards/provisioner.yaml

Reply:
+1